File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageImage.java
package com.dhsdevelopments.potato.clientapi.message;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class MessageImage implements Serializable
{
    @SerializedName( "file" )
    public String file;

    @SerializedName( "width" )
    public int width;

    @SerializedName( "height" )
    public int height;
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageImage.kt
package com.dhsdevelopments.potato.clientapi.message

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class MessageImage : Serializable {
    @SerializedName("file")
    lateinit var file: String

    @SerializedName("width")
    var width: Int = 0

    @SerializedName("height")
    var height: Int = 0
}
